### PR TITLE
PATAND-267: Update other consent layout

### PR DESCRIPTION
**Pull request by jaimesDSG**
## Objective
Implement the change for the completion step

## Branches
- PAT: task/PATAND-267_Update_consent_layout
- Axon: task/PATAND-267_Update_consent_layout
- RS: task/PATAND-267_Update_consent_layout
- Cortex: development

#### Credentials:
- Environment: QA
- Org: hybridstudy
- Study: QA-Regression
- New user

#### Steps:
* open flask/ resumed it 
* Start as a new user 
* Start the consent task
Expected: 
* The new UIs will fit these [zeplin](https://app.zeplin.io/project/5dc576e8f5b84283ac09b02a/screen/5e1cb02264fab7802e7d54d1) 

Close PATAND-267

### CODE REVIEW

File: backbone/src/main/java/org/researchstack/backbone/ui/step/body/DecimalQuestionBody.java [hunk 1]
```diff
@@ -32,6 +34,7 @@
     //-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
     private int viewType;
     private EditText editText;
+    private TextInputLayout textEntry;
```
Comment (MohamedSaleh-medable) [inline backbone/src/main/java/org/researchstack/backbone/ui/step/body/DecimalQuestionBody.java:37]: This variable not used

**Comment by Gryzor** [comment]
I noticed a few inconsistencies. Comments incoming.

**Comment by Gryzor** [comment]
First Consent Page. Expand the image below to see it. I'm always comparing [to this zeplin compo](https://app.zeplin.io/project/5dc576e8f5b84283ac09b02a/screen/5e1cb02264fab7802e7d54d1) and similar ones.

<details>

Screenshot_1593605068

</details>

1. First thing I noticed was the colors. The Theme for the consent is the generic teal, not BLUE like the rest of the app.
2. the `Learn more` is not correctly upper-case like in the Zeplin.
3. The Next button appears to be a secondary color in Zeplin (same as learn more) yet in the app I see it dark blue.

Later on...
<details>

Screenshot_1593605335

</details>

Here I see the Theme issue. The RadioButtons are themed aquamarine/teal, they should respect the primary theme of the app. This theme issue is observed across the consent so I won't repeat it.

<details>

Screenshot_1593605427

</details>

In this supposed to look like that with the gray font + gray background?

<details>

Screenshot_1593605565

</details>

1. ^ This doesn't look/behave like a true MaterialDesign component. I was expecting the "First Name" to be the hint until the box had focus and then move it up, like the control does by design. The text "answer" is redundant and shouldn't even be there. 

2. There's a separator between both text boxes. 

I've compared against the [zeplin comp for this screen](https://app.zeplin.io/project/5dc576e8f5b84283ac09b02a/screen/5e1cb022b8fe9fbccdeb53cc). I still think that comp is wrong, the "hint" (answer) shouldn't be there. The hint should be First Name and Last Name and they should look like that when you tap on them (sans the "answer" text). 

<details>

Screenshot_1593605840

</details>

^^ The border is too thick compared to the zeplin. The paddings in Sign here [are not the same](https://app.zeplin.io/project/5dc576e8f5b84283ac09b02a/screen/5e1cb022f3947dbdbfcf2d5c), and the CLEAR button once you sign, is not centered [like in the comp](https://app.zeplin.io/project/5dc576e8f5b84283ac09b02a/screen/5e1cb022f523ba7ad54608cc).

Other than that, it worked fine. :)

**Comment by jaimesDSG** [comment]
> First Consent Page. Expand the image below to see it. I'm always comparing [to this zeplin compo](https://app.zeplin.io/project/5dc576e8f5b84283ac09b02a/screen/5e1cb02264fab7802e7d54d1) and similar ones.
> 
> 1. First thing I noticed was the colors. The Theme for the consent is the generic teal, not BLUE like the rest of the app.
> 2. the `Learn more` is not correctly upper-case like in the Zeplin.
> 3. The Next button appears to be a secondary color in Zeplin (same as learn more) yet in the app I see it dark blue.

The consent header now is display with the primary color of the step.
The Learn more (and all the more info button) will be on upper-case
The next button is on Dark blue as all the other button on the submit bar. I didn't change anything, it's part of the common submit bar.

> 
> Later on...
> 
> Here I see the Theme issue. The RadioButtons are themed aquamarine/teal, they should respect the primary theme of the app. This theme issue is observed across the consent so I won't repeat it.

I didn't change the whole theme issue, we didn't have any zeplin for it so i let it as it is :/

> 
> In this supposed to look like that with the gray font + gray background?

I change the background because it didn't fit the rest of the application but i let the gray front because it was actually something that change when the value is selected (the value selected is in black, the other are in grey).

> 
> 1. ^ This doesn't look/behave like a true MaterialDesign component. I was expecting the "First Name" to be the hint until the box had focus and then move it up, like the control does by design. The text "answer" is redundant and shouldn't even be there.
> 2. There's a separator between both text boxes.

This is also the reason why i did this like that. It's more compliant with the rest of the app ( for example the autocomplete step is done like this : [zeplin](https://app.zeplin.io/project/5dc576e8f5b84283ac09b02a/screen/5dcaf47f0425621df0f56380))
The separator is here because we use a formStep for this step and all the "substep" have a separator between them. I prefer to not remove to and break some other layout on another part.

> 
> I've compared against the [zeplin comp for this screen](https://app.zeplin.io/project/5dc576e8f5b84283ac09b02a/screen/5e1cb022b8fe9fbccdeb53cc). I still think that comp is wrong, the "hint" (answer) shouldn't be there. The hint should be First Name and Last Name and they should look like that when you tap on them (sans the "answer" text).
> 

Totally agree with that but we never use this hint behaviour anywhere on the app so i did as usual.

> ^^ The border is too thick compared to the zeplin. The paddings in Sign here [are not the same](https://app.zeplin.io/project/5dc576e8f5b84283ac09b02a/screen/5e1cb022f3947dbdbfcf2d5c), and the CLEAR button once you sign, is not centered [like in the comp](https://app.zeplin.io/project/5dc576e8f5b84283ac09b02a/screen/5e1cb022f523ba7ad54608cc).
> 

This is updated and normally it will be fine 🤞 

> Other than that, it worked fine. :)

Great :D 

PS: I will discuss with Felix about the radio button and the lastName/firstname part

**Pull request merged**